File: beginner/java/2510.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n;
        String bandido;

        n = scanner.nextInt();

        for(int i = 0; i < n; i++){
            bandido = scanner.next();
            System.out.printf("Y%n");
        }
    }
}

File: beginner/c/2510.c
#include <stdio.h>

int main() {
    int n;
    char bandido[100];

    scanf("%d", &n);

    for(int i = 0; i < n; i++){
        scanf("%s", bandido);
        printf("Y\n");
    }

    return 0;
}
